storey/aggregations.py

```diff
@@ -251,9 +251,17 @@ def __init__(self, features: List[str], table: Union[Table, str], key: Union[str
                  fixed_window_type: Optional[FixedWindowType] = FixedWindowType.CurrentOpenWindow, **kwargs):
         self._aggrs = []
         self._enrich_cols = []
+        self._table = None
         resolved_aggrs = {}
+        Flow.__init__(self, **kwargs)
+        if isinstance(table, str):
+            if not self.context:
+                raise TypeError("Table can not be string if no context was provided to the step")
+            self._table = self.context.get_table(table)
```

Comment:
This code is already in the super class. Afaict, this will cause the table to be instantiated and then cloned 17 lines below, which doesn't make much sense.